Add chapi-http example (#20)
w3c-ccg/vc-examples: docs/chapi-http-edu/README.md

      // that will be attached to the VerifiablePresentation response
      challenge: "99612b24-63d9-11ea-b99f-4f66f3e4f81a",
      // the domain that must be digitally signed in the authentication
      // proof that will be attached to the VerifiablePresentation
      // response, identifying the recipient
      domain: "issuer.interop.transmute.world"
    }
  }
};
const result = await navigator.credentials.get(query);
```

Example Response:

```json
{
  "@context": "https://www.w3.org/2018/credentials/v1",
  "type": "VerifiablePresentation",
  "holder": "did:key:z6MkjRagNiMu91DduvCvgEsqLZDVzrJzFrwahc4tXLt9DoHd",
  "proof": {
    "type": "Ed25519Signature2018",
    "created": "2020-04-02T18:21:44Z",
    "verificationMethod": "did:key:z6MkjRagNiMu91DduvCvgEsqLZDVzrJzFrwahc4tXLt9DoHd#z6MkjRagNiMu91DduvCvgEsqLZDVzrJzFrwahc4tXLt9DoHd",
    "proofPurpose": "authentication",
    "challenge": "99612b24-63d9-11ea-b99f-4f66f3e4f81a",
    "domain": "issuer.interop.transmute.world",
    "jws": "eyJhbGciOiJFZERTQSIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..2xjpkHW6EY-cKD8DrMIkkiB2Q_k6kHynTbR7XGgtYR92blQWpL6Q-2nTdQi1rNhJtmHw1wWWssKMO0EdIEnsCw"
  }
}
```

## 2. Issuance

> Next the DIDAuth Response and a request for a University degree need to made to the university website.

Reviewer: ```suggestion
Next a request for a University degree needs to be made to the university website.
```

Normally, a website would first verify the DID Auth, and then contruct the request for a credential to be issued against the HTTP Issuer API.

> For demonstration purposes, we simply request the credential from the issuer api directly. Other mechanisms migth be used to facilitate this transfer in the future, such as DIDComm. For example:

Reviewer: Why should we skip the DIDAuth response validation in the demo?

Author: Good point, i will include an example of verifying it.

User Agent -> Issuer Website (university.example.com) -> Internal Issuer API

> For now we will use HTTP and CHAPI to request the credential be issued and stored in the user's wallet directly.

Reviewer: > we

Who? The user's agent or the issuer?

Author: This was worded poorly... I have updated it. Essentially, I'm trying to work around not having the website "university.example.com" in the flow, since it just forwards requests to the standard HTTP APIs...
